cpp/benchmarks/common/generate_input.cu

```diff
@@ -175,18 +231,30 @@ struct random_value_fn<T, std::enable_if_t<cudf::is_fixed_point<T>()>> {
     : lower_bound{desc.lower_bound},
       upper_bound{desc.upper_bound},
       dist{make_distribution<rep>(desc.id, desc.lower_bound, desc.upper_bound)}
-  {
-  }
-
-  T operator()(std::mt19937& engine)
   {
     if (not scale.has_value()) {
       int const max_scale = std::numeric_limits<rep>::digits10;
-      auto scale_dist     = make_distribution<int>(distribution_id::NORMAL, -max_scale, max_scale);
-      scale = numeric::scale_type{std::max(std::min(scale_dist(engine), max_scale), -max_scale)};
+      std::binomial_distribution<int> scale_dist_normal(2 * max_scale, 0.5);
```

Comment:
Why do we want a binomial distribution here? If there is a particular reason, then we should probably rename this struct to reflect the distribution or at least update the documentation. My default expectation for a random value with no priors is a uniform distribution.

Unrelated, the `brief` for this struct is incorrect and claims that it isn't implemented yet.

Reply:
In old code, `make_distribution<int>(distribution_id::NORMAL` returned `binomial_distribution`. So, I replaced it here.
I don't know the rationale behind the choice of distribution.  @vuule https://github.com/rapidsai/cudf/pull/9776 could comment on this.
uniform should be good here.

Reply:
Real-world data more commonly has normal distribution (citation needed :) ), rather than uniform. This is why it's the default. The encoding/compression of formats supported in cuIO is affected by the data distribution (more value around zero vs uniform/large values) so I aimed to generate "realistic" data here for the benchmarks.
I'm open to changing the defaults, but IMO it should be done in a separate PR.

Reply:
Ohhh OK so the goal here was not to estimate a binomial distribution with a normal (indicating that we think a binomial distribution is a good real-world model), but rather to estimate a normal with a binomial? That is very backwards of anything that I'm used to seeing, but if it's an API limitation on existing available distributions for integers that makes a whole lot more sense.

I do not think that we need to change the choice of distribution in this PR, but assuming my new understanding is correct I do think the comment needs to be updated to reflect that our target distribution is normal, not binomial.

Reply:
That's right. We need normal integer distribution. everything coded here is just approximation because thrust does not have normal distribution for integers yet.

Reply:
Got it. Could you still add a comment indicating what's going on here? The normal approximation to the binomial is almost always used the other way around because evaluating a binomial distribution (specifically, evaluating n choose k) for large n/k becomes prohibitively expensive and impossible to do by hand. In my experience going the other way and approximating a normal with a binomial is very nonstandard. de Moivre's theorem (and any other version of CLT) are in general applied to justify estimating other distributions by a normal distribution, not the other way around.